sdk/loadtestservice/azure-mgmt-loadtestservice/tests/test_loadtestresource_operations.py

```diff
@@ -0,0 +1,130 @@
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+
+import unittest
+
+from azure.mgmt.loadtestservice import LoadTestMgmtClient
+from azure.mgmt.loadtestservice.models import LoadTestResource, LoadTestResourcePatchRequestBody, QuotaBucketRequest
+from devtools_testutils import AzureMgmtRecordedTestCase, recorded_by_proxy, ResourceGroupPreparer
+
+AZURE_LOCATION = 'westus2'
+LOAD_TEST_RESOURCE_NAME = 'py-sdk-testing-resource-cp'
+
+class TestLoadTestResourceOperations(AzureMgmtRecordedTestCase):
+    
+    def setup_method(self, method):
+        self.loadtestservice_client = self.create_mgmt_client(
+            LoadTestMgmtClient
+        )
+    
+    @ResourceGroupPreparer()
+    @recorded_by_proxy
+    def test_load_test_resource_operations(self, resource_group):
+
+        # Create a load test resource
+        loadtestresource_create_payload = LoadTestResource(location=AZURE_LOCATION)
+        
+        # Create a load test resource create begin - returns a poller
+        loadtest_resource_poller = self.loadtestservice_client.load_tests.begin_create_or_update(resource_group.name, LOAD_TEST_RESOURCE_NAME, loadtestresource_create_payload)
+        
+        # Wait for the poller to complete
+        while loadtest_resource_poller.done()==False:
+            loadtest_resource_poller.wait(2)
+        
+        # Get the result of the poller
+        loadtest_resource = loadtest_resource_poller.result()
+        
+        # Assert the result of the poller
+        assert loadtest_resource != None
+        assert loadtest_resource.name == LOAD_TEST_RESOURCE_NAME
+        assert loadtest_resource.location == AZURE_LOCATION
+        assert loadtest_resource.id != None
+        
+        # Get the load test resource
+        loadtest_resource_get = self.loadtestservice_client.load_tests.get(resource_group.name, LOAD_TEST_RESOURCE_NAME)
+        
+        # Assert the result of the get operation
+        assert loadtest_resource.name == loadtest_resource_get.name
+        assert loadtest_resource.location == loadtest_resource_get.location
+        assert loadtest_resource.id == loadtest_resource_get.id
+
+
+        # Update the load test resource
+        loadtestresourcePatchdata = LoadTestResourcePatchRequestBody(
+            identity={
+                'type': 'SystemAssigned'
+            }
+        )
+
+        # load test resource update begin - returns a poller
+        loadtest_resource_patch_poller = self.loadtestservice_client.load_tests.begin_update(resource_group.name, LOAD_TEST_RESOURCE_NAME, loadtestresourcePatchdata)
+        
+        # Wait for the poller to complete
+        while loadtest_resource_poller.done()==False:
+            loadtest_resource_poller.wait(2)
```

Comment:
same comment: https://github.com/Azure/azure-sdk-for-python/pull/26842/files#r996735451